server/src/test/java/org/elasticsearch/snapshots/DedicatedClusterSnapshotRestoreIT.java

```diff
@@ -928,7 +928,6 @@ public void testMasterAndDataShutdownDuringSnapshot() throws Exception {
         }
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/25281")
```

Comment:
Can we have two separate PRs - for #41581 and #25281?

Reply:
Sure, revered the related changes will open a follow up PR then